src/components/Avatar/Avatar.stories.tsx

```diff
@@ -25,3 +25,23 @@ AvatarLetters.args = {
     theme: 'letters',
     text: 'DM',
 };
+
+export const RoundedAvatarImageDefault = Template.bind({});
+RoundedAvatarImageDefault.args = {
+    theme: 'image',
+    rounded: true,
```

Comment:
for a boolean please consider using a question.
EG: `isRounded` or `hasRounded`

we have tried to use this standard in the kit but if you see any we missed please point them out or better yet fix them please =]